tutorials/pretraining-data-curation/red-pajama-v2-curation-tutorial.ipynb

```diff
@@ -804,8 +804,9 @@
    "source": [
     "from nemo_curator.modules import ExactDuplicates\n",
     "\n",
-    "def pre_imports():\n",
-    "    import cudf  # noqa: F401"
+    "\n",
+    "def pre_imports() -> None:\n",
+    "    pass"
```

Comment:
Is this still needed?

Reply:
I have added `import cudf` back, good catch on that